src/backend/binaryen/wasm_expr_gen.ts

```diff
@@ -770,6 +770,17 @@ export class WASMExpressionGen {
             !FunctionalFuncs.treatAsAny(leftValueType.kind) &&
             FunctionalFuncs.treatAsAny(rightValueType.kind)
         ) {
+            // the positions of the l-operand and r-operand are interchanged, and binary operators need to be reversed.
```

Comment:
What's the meaning of `the positions of the l-operand and r-operand are interchanged`?